public/js/modules.js

```diff
@@ -576,6 +576,117 @@ SevenSegDisplay.prototype.customDraw = function() {
     ctx.stroke();
 }
 
+function SixteenSegDisplay(x, y, scope = globalScope) {
```

Comment:
Let's put this into its own file, in which case you'll update a couple of other files to include it.
See #199

Reply:
@fpintos let us not do this as of yet for non-critical modules like these. I know it is better to have these in separate files and then bundle them together, wouldn't it be better to separate them once we setup webpack and other tools?